static/src/javascripts/projects/commercial/modules/dfp/define-slot.ts

```diff
@@ -1,20 +1,24 @@
-/**
- * @typedef {import('@guardian/commercial-core').AdSize} AdSize
- * @typedef {import('@guardian/commercial-core').SizeMapping} SizeMapping
- */
+import type { SizeMapping, SlotName } from '@guardian/commercial-core';
 import { once } from 'lodash-es';
-import config from '../../../../lib/config';
+import type { IasPETSlot, IasTargeting } from 'types/ias';
 import { breakpoints } from '../../../../lib/detect';
```

Comment:
If you use `detect-viewport.ts` from libs, you’ll get much better typings!

Reply:
sweet, thanks!

Reply:
unfortunately the widths are different in `detect-viewport.ts`? 🤔

Reply:
I thought I had confirmed they were identical using the following test: https://github.com/guardian/frontend/blob/main/static/src/javascripts/lib/detect-viewport.spec.ts

Probably not thorough enough? The new API uses Source so will definitely be in line with DCR widths.